app/models/push_notification.rb

```diff
@@ -0,0 +1,5 @@
+class PushNotification < ActiveRecord::Base
+  belongs_to :event, foreign_key: 'event_id'
+
+  validates_presence_of :event
```

Comment:
Please, use the (latest) [validates](http://guides.rubyonrails.org/active_record_validations.html#presence) syntax instead. We don't know how long the old syntax will be maintained.